docs/contributor/resources/02-manifest.md

```diff
@@ -12,7 +12,7 @@ kubectl get crd manifests.operator.kyma-project.io -o yaml
 
 ## Patching
 
-The [Runner](../../../pkg/module/sync/runner.go) is responsible for creating and updating Manifest CRs using Server Side Apply (SSA). An update is only performed when one of the following conditions is met:
+Responsible for creating and updating Manifest CRs is the so-called Runner. It is using Server Side Apply (SSA) where an update is only performed when one of the following conditions is met:
```

Comment:
The struct is called runner and also the file. So should be easy to find without direct link